app/src/main/res/drawable/ic_launcher_monochrome.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="108dp"
+    android:height="109dp"
+    android:viewportWidth="108"
+    android:viewportHeight="109">
+  <group>
+    <clip-path
+        android:pathData="M25.75,25.93h56.5v58.25h-56.5z"/>
+    <path
+        android:pathData="M80.22,45.46C78.99,42.51 76.5,39.33 74.54,38.32C75.94,41.01 76.9,43.9 77.41,46.89L77.41,46.93C74.21,38.98 68.79,35.78 64.36,28.79C64.13,28.43 63.91,28.07 63.69,27.71C63.58,27.52 63.48,27.33 63.38,27.13C63.2,26.77 63.06,26.4 62.96,26.01C62.96,25.99 62.95,25.98 62.94,25.96C62.93,25.95 62.91,25.94 62.89,25.94C62.88,25.93 62.86,25.93 62.84,25.94C62.84,25.94 62.83,25.94 62.83,25.95C62.82,25.95 62.81,25.95 62.81,25.96L62.82,25.94C56.53,29.61 53.92,36.05 53.25,40.14C51.31,40.25 49.4,40.74 47.64,41.57C47.47,41.65 47.34,41.8 47.27,41.98C47.21,42.16 47.22,42.36 47.3,42.53C47.38,42.71 47.52,42.84 47.7,42.91C47.88,42.97 48.08,42.96 48.25,42.88C49.79,42.16 51.44,41.73 53.13,41.61L53.3,41.6C53.53,41.59 53.76,41.58 53.99,41.57C55.37,41.56 56.73,41.75 58.05,42.14C58.13,42.16 58.21,42.18 58.28,42.21C58.5,42.28 58.72,42.35 58.94,42.43C59.09,42.49 59.25,42.55 59.4,42.61C59.53,42.66 59.66,42.71 59.78,42.77C59.98,42.85 60.17,42.94 60.36,43.04C60.45,43.08 60.54,43.12 60.62,43.17C60.81,43.27 60.99,43.37 61.17,43.47C61.29,43.54 61.41,43.61 61.52,43.68C63.57,44.94 65.27,46.7 66.47,48.79C64.47,47.41 62.05,46.82 59.65,47.13C69.83,52.21 67.09,69.68 52.99,69.02C51.74,68.97 50.5,68.73 49.31,68.32C49.04,68.21 48.76,68.1 48.48,67.98C48.32,67.91 48.16,67.83 48.01,67.75C44.55,65.97 41.7,62.6 41.34,58.51C41.34,58.51 42.65,53.66 50.69,53.66C51.56,53.66 54.05,51.24 54.1,50.54C54.09,50.31 49.16,48.35 47.24,46.47C46.22,45.46 45.73,44.97 45.3,44.61C45.07,44.41 44.82,44.23 44.57,44.06C43.92,41.81 43.89,39.42 44.49,37.16C41.88,38.42 39.56,40.2 37.68,42.41H37.67C36.54,40.99 36.62,36.32 36.68,35.34C36.35,35.47 36.03,35.64 35.74,35.84C34.75,36.54 33.82,37.34 32.97,38.2C32.01,39.18 31.12,40.24 30.33,41.37V41.37V41.37C28.51,43.94 27.22,46.84 26.53,49.92C26.35,50.78 26.19,51.64 26.07,52.52C26.04,52.74 26.01,52.96 25.98,53.18C25.9,53.72 25.85,54.25 25.82,54.79C25.82,54.82 25.82,54.85 25.81,54.88C25.79,55.19 25.77,55.5 25.75,55.82V55.96C25.82,63.05 28.51,69.86 33.3,75.06C38.1,80.25 44.65,83.47 51.67,84.06C58.7,84.66 65.7,82.6 71.29,78.28C76.88,73.97 80.68,67.71 81.92,60.74C81.97,60.37 82.01,60.01 82.05,59.64C82.63,54.84 82,49.96 80.21,45.47L80.22,45.46Z"
```

Comment:
Android studio is complaining about this being a long vector path. I'm not immediately sure what we do to optimize them. @gabrielluong what's the best practice here?

Reply:
We have a bunch icons that will have the same complaint. Based on the discussions with UX we have had, it would be up to them to optimize it for us when they secure a single source of truth for all the icons and other design system related items. For now, I would safely just ignore it the complaint.